syntax = "proto3";

package PFDESolverServer;

option go_package = "/protos";

import "config.proto";
import "result.proto";

service TFDESolverServer {
    rpc RunTask(PFDESolver.TClientConfig) returns (PFDESolver.TResults) {}
}
